pkg/test/resources/apis/sample/sample_types.go

```diff
@@ -52,10 +52,19 @@ func (t *TrafficRouteResource) Validate() error {
 var _ model.ResourceList = &TrafficRouteResourceList{}
 
 type TrafficRouteResourceList struct {
+	Total      uint32
 	Items      []*TrafficRouteResource
 	Pagination model.Pagination
 }
 
+func (l *TrafficRouteResourceList) GetTotal() uint32 {
+	return l.Total
+}
+
+func (l *TrafficRouteResourceList) SetTotal(total uint32) {
+	l.Total = total
+}
+
```

Comment:
those 2 funcs and Total in struct is not needed anymore, right?

Reply:
Cleaned, thanks!